lib/offline/indexeddb/v1_storage_cell.js

```diff
@@ -218,6 +219,7 @@ shaka.offline.indexeddb.V1StorageCell = class
       mimeType: old.mimeType,
       codecs: old.codecs,
       frameRate: old.frameRate,
+      pixelAspectRatio: old.pixelAspectRatio,
```

Comment:
The V1 format didn't have this value, so pixelAspectRatio: null here should suffice.

Reply:
I changed it!